radix-engine-common/src/types/entity_type.rs

```diff
@@ -2,33 +2,108 @@ use strum::FromRepr;
 
 //=========================================================================
 // Please update REP-60 after updating types/configs defined in this file!
+// Please use and update REP-71 for choosing an entity type prefix
 //=========================================================================
 
 /// An enum which represents the different addressable entities.
 #[repr(u8)]
 #[derive(Clone, Copy, Debug, PartialEq, Eq, Ord, PartialOrd, FromRepr)]
 pub enum EntityType {
-    GlobalPackage,
-    GlobalFungibleResource,
-    GlobalNonFungibleResource,
-    GlobalEpochManager,
-    GlobalValidator,
-    GlobalClock,
-    GlobalAccessController,
-    GlobalAccount,
-    GlobalIdentity,
-    GlobalGenericComponent, // generic
-
-    GlobalVirtualEcdsaAccount,
-    GlobalVirtualEddsaAccount,
-    GlobalVirtualEcdsaIdentity,
-    GlobalVirtualEddsaIdentity,
-
-    InternalFungibleVault,
-    InternalNonFungibleVault,
-    InternalAccount,
-    InternalKeyValueStore,
-    InternalIndex,
-    InternalSortedIndex,
-    InternalGenericComponent, // generic
+    //=========================================================================
+    // Package (start with char p for package)
+    //=========================================================================
+    /// A global package entity. Gives Bech32 prefix: `p` followed by one of `5`, `4`, `k` or `h`.
+    GlobalPackage = 0b00001101, //------------------- 00001 => p, 101xx => 54kh [pkg vanity prefix]
+
+    //=========================================================================
+    // System Components (start with char s for system)
+    //=========================================================================
+    /// The global epoch manager entity. Gives Bech32 prefix: `s` followed by one of `c`, `e`, `6` or `m`.
+    GlobalEpochManager = 0b10000110, //-------------- 10000 => s, 110xx => ce6m [se vanity prefix]
+
+    /// A global validator entity. Gives Bech32 prefix: `s` followed by one of `v`, `d`, `w` or `0`.
+    GlobalValidator = 0b10000010, //----------------- 10000 => s, 110xx => vdw0
+
+    /// The global clock entity. Gives Bech32 prefix: `s` followed by one of `c`, `e`, `6` or `m`.
```

Comment:
Tiny thing, but I believe it should be “`5`, `4`, `k` or `h`” here

Reply:
Spot on, thanks! Fixed :)